developer_manual/scripts/read-config.php

```diff
@@ -77,13 +77,27 @@ function findPath()
  * @see https://secure.php.net/manual/en/reserved.variables.argv.php
  */
 if (count($argv) != 2) {
-    echo "Need to provide the path to the ownCloud directory.";
-    exit(1);
+	echo "Command usage: read-config.php <full path to ownCloud root dir> \n";
+	echo "Please provide the path to the ownCloud directory. \n";
+	exit(1);
 }
 
-$ownCloudConfigFile = sprintf("%s/config/config.php", (string) $argv[1]);
+// create a realpath and remove trailing "/" from argument if present
+$ocRoot = rtrim( (string) $argv[1], "/");
+$ownCloudConfigFile = sprintf("%s/config/config.php", $ocRoot);
 
-if (file_exists($ownCloudConfigFile)) {
+if (!realpath($ownCloudConfigFile)) {
+	// if path/file does not exist, return an error message
+	echo 'File not found: ' . $ownCloudConfigFile . PHP_EOL;
+} else {
+	// return the path, identified by a leading "/" and no new line character at the end
     require_once($ownCloudConfigFile);
-    print (new SimpleConfigReader($CONFIG))->findPath();
+    $result = (new SimpleConfigReader($CONFIG))->findPath($ocRoot . '/apps');
```

Comment:
See comments in https://github.com/owncloud/documentation/pull/4314/files#r205150170 (pls).